rest/bulk_api.go

```diff
@@ -27,6 +27,11 @@ import (
 
 // HTTP handler for _all_docs
 func (h *handler) handleAllDocs() error {
+
+	if !db.IsAllDocsIndexCreated {
```

Comment:
May want to just base this on enableStarChannel, to avoid scenarios where the index gets deleted by user after we've done the startup check for existence.